src/controllers/user.controller.ts

```diff
@@ -0,0 +1,85 @@
+import {Request, Response} from 'express';
+import db from '../database/sequelize';
+import {getAutoSuggestUsers} from '../services/user.service';
+const User = db.user;
+
+class UserController {
+  static readonly DEFAULT_USERS_LIMIT = 10;
+  static readonly LOGIN_SUBSTRING = '';
+
+  async getUsers(req: Request, res: Response): Promise<any> {
+    User.findAll({raw: true})
+    .then(users => {
+      const filteredUsers = getAutoSuggestUsers(users as any[], UserController.LOGIN_SUBSTRING, UserController.DEFAULT_USERS_LIMIT);
+
+      return res.json(filteredUsers);
+    })
+    .catch(err => console.log(err));
+  };
+
+  async getUserById(req: Request, res: Response): Promise<any> {
+    const id = req.params.id;
+
+    User.findByPk(id)
+    .then(user => {
+      if(!user) return;

```

Comment:
You should return a response to a client, for example: `res.status(404).json({ message: 'Not found' })`